backend/src/api/routes/deployments/index.routes.ts

```
@@ -48,6 +52,66 @@ router.post('/', verifyAdmin, async (req: AuthRequest, res: Response, next: Next
+        throw new AppError(
+          "Zip file is required",
+          400,
+          ERROR_CODES.INVALID_INPUT
+        );
+      }
+
+      // 1️⃣ Create deployment
+      const deployment = await deploymentService.createDeployment();
+
+      // 2️⃣ Save uploaded zip
+      const deploymentsDir = path.join("/tmp", "deployments");
```

ESLint

[error] Replace `"/tmp",·"deployments"` with `'/tmp',·'deployments'`

(prettier/prettier)